README.md

```diff
@@ -96,20 +96,20 @@ After you're finished please remove all the comments and instructions!
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
+Key Features
+
+key features of the application:
+ The app allows user to update information.
 
-> Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+
 
 - [Live Demo Link](https://yourdeployedapplicationlink.com)
```

Comment:
- [ ] [OPTIONAL] The key feature section is void of any meaningful key feature. This is important for your future projects in Microverse as it is an essential component of a professional README file. 
- [ ] You should always remove the demo section whenever that project does not require deploying. 


![image](https://user-images.githubusercontent.com/84156714/219361790-90b7a386-6276-47df-ba9a-64252ca1a41e.png)